src/metabase-report-executor/src/main/java/com/champtitles/metabasereportexecutor/executor/App.java

```diff
@@ -32,6 +32,8 @@ public String handleRequest(Object event, Context context) {
         logger.info("running query for card: {}", metabaseCardId);
         byte[] xlsxBody = metabaseClient.queryCardGetXlsx(metabaseCardId);
         s3Writer.uploadXlsx(xlsxBody, "card" + metabaseCardId);
+
+        logger.info("done processing card: {}", metabaseCardId);
```

Comment:
This is baseline OK, but should always consider if more information may be useful, for example, some info about the results, e.g., size of file. No changes required.

Reply:
@dcrough-ownum makes sense and I agree. I primarily added this as a sentinel for integration testing to validate the lambda was successful